ci/recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
 
   run:
     - python {{ python }}
-    - cutadapt
+    - cutadapt >=2,<3
```

Comment:
I think you might have a bit of merge lint here - the new cutadapt release should be `>=3`